### PR TITLE
forms/templates/form//form_field: restructure form to conforn to a11y…

### LINKED ISSUE #260: Screenreader does not read title of fields of contact and participation form
**URL:** https://digitalstrategie-dev.liqd.net/kontakt/
**user:** participant with screenreader
**expected behaviour:** screenreader reads fields of fields in order for people using screen reader to know where to put in their name
**behaviour:** the headline of fields are not being read out - 
**important screensize:**desktop
**device & browser:** macOS Monterey, using VoiceOver as Screenreader
**Comment/Question:** I am not very well trained in this so I am unsure if I got it right...or how are people able to know what kind of field they are in?

Screenshot?

### LINKED ISSUE #263: #5295 Check & adapt participation and contact form to new grid
**URL:** https://digitalstrategie-dev.liqd.net
**important screensize:** *Both*
**device & browser:** *Safari & Chrome*
**Comment/Question:** *
1 – There is a black dot behind the Error messages
2 – There is also a black dot in each fields of action option
3 – Can we make the space between "Fields of Action" and "Required fields" * 

Screenshot 2022-02-15 at 07 27 51
Screenshot 2022-02-15 at 07 30 50

### PR DESCRIPTION
**Pull request by phillimorland**
… standards and ensure screen readers work and style fixes #263 fixes #260

### CODE REVIEW

**Comment by phillimorland** [comment]
i'm not totally sure on the error handling for the multiselect field but unless we want to make an entierly new widget I think this will be fine, it was also wrong before anyway

**Comment by phillimorland** [comment]
oupps hold off a min

File: apps/forms/templates/apps_forms/includes/form_field.html [hunk 1]
```diff
@@ -1,8 +1,7 @@
 {% load i18n widget_tweaks form_field_tags %}
 
 {% if field|is_checkbox %}
-  <div class="form-group">
-    <div class="form-check">
+  <div class="form-group form-check">
```
Comment (khamui) [inline apps/forms/templates/apps_forms/includes/form_field.html:4]: so good!

**Comment by khamui** [comment]
nice work!
looking good semantically (cannot test a11y @phillimorland happy to chat how you approach it) and very nice refactoring!

**Pull request merged**